nodebuilder/p2p/flags.go

```diff
@@ -2,14 +2,20 @@ package p2p
 
 import (
 	"fmt"
+	"os"
+	"strings"
 
 	"github.com/multiformats/go-multiaddr"
 	"github.com/spf13/cobra"
 	flag "github.com/spf13/pflag"
 )
 
-var (
-	p2pMutualFlag = "p2p.mutual"
+const (
+	// DefaultNetwork is the default network of the current build.
+	DefaultNetwork   = Arabica
+	EnvCustomNetwork = "CELESTIA_CUSTOM"
+	nodeNetworkFlag  = "node.network"
```

Comment:
```suggestion
	nodeNetworkFlag  = "p2p.network"
```

is the goal of keeping it `node.network` for it to not be breaking? I think i'd rather break CLI now and make it consistent.

Reply:
done